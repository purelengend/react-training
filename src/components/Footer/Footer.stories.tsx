import type { Meta, StoryObj } from '@storybook/react';
import Footer from './index';
import './Footer.module.css';

const meta: Meta<typeof Footer> = {
  title: 'Example/Footer',
  component: Footer,
};

type Story = StoryObj<typeof Footer>;

export const FooterExample: Story = {};
export default meta;
