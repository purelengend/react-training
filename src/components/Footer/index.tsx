import footerStyles from '@components/Footer/footer.module.css';

const Footer = () => {
  return <footer className={footerStyles['footer-container']}></footer>;
};

export default Footer;
