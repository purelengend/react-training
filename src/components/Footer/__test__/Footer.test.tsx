import { renderer } from '@src/test/test-utils';

import Footer from '..';

describe('Footer test case', () => {
  it('should render correctly', () => {
    const mockFooter = renderer(<Footer />);

    expect(mockFooter.toJSON()).toMatchSnapshot();
  });
});
