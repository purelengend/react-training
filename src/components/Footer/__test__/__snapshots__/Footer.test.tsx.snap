// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer test case > should render correctly 1`] = `
<footer
  className="_footer-container_1a5889"
/>
`;
